config/environments/test.rb

```diff
@@ -11,24 +11,20 @@
   # While tests run files are not watched, reloading is not necessary.
   config.enable_reloading = false
 
-  # Turn false under Spring and add config.action_view.cache_template_loading = true.
```

Comment:
It's not necessary as [it's the opposite of config.enable_reloading](https://guides.rubyonrails.org/configuring.html#config-cache-classes) defined above.